sdk/core/core-lro/package.json

```diff
@@ -102,13 +99,10 @@
   },
   "devDependencies": {
     "@azure/core-http": "^2.0.0",
```

Comment:
Can we remove core-http from the devDependencies?

Reply:
This is out of scope for this PR but I agree we should remove it. However doing so will require rewriting test files. I can do that in another PR though.